data/itemGenerator.js

```diff
@@ -39,12 +40,12 @@ function generateAttendeeInstructions(golden) {
 const ageRanges = {
   "Adult": {
     "type": "QuantitativeValue",
-    "maxValue": 18,
+    "minValue": 18,
```

Comment:
This was just a bug. Noticed by my editor

Reply:
Noice